### PR TITLE
Added svg icon

### PR DESCRIPTION
**Pull request by sboriskirov**
Added svg logo for the header

### CODE REVIEW

File: src/images/Fluiditype-logo.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        from="3.5" 
+        to="7" 
+        dur="2s" 
+        repeatCount="4"
+        begin="baseTwo.click"            
+        values="3.5; 7; 3.5;"
+        keyTimes="0 ; 0.2 ; 1;"
+        keySplines="0.99 0 0.47 0.42;"
+    />
+</svg>
```
Comment (karakashevivo) [inline src/images/Fluiditype-logo.svg:24]: Optimise the `svg` [here](https://jakearchibald.github.io/svgomg/), the  upload again.

File: src/components/content-reverse.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import herofluiditype from "../images/hero-fluiditype.gif"
 
 const ContentReverse = () => (
   <div>
```
Comment (karakashevivo) [inline src/components/content-reverse.js:10]: It's a bad practise to wrap content with an extra `div`, you can easy delete it without consequences.

File: src/components/content-reverse.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ import herofluiditype from "../images/hero-fluiditype.gif"
...
-    <content>
+    <content class="center-second">
```
Comment (karakashevivo) [inline src/components/content-reverse.js:11]: `<content>` [is a depriccated HTML tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content).  You can use simple `div` or `section`.

File: src/styles/styles.css [hunk 4]
```diff
@@ -20,35 +20,68 @@ content.center{
     text-align: center;
     align-content: center;
     grid-template-columns: 70vw;
-    min-height: calc(210vh - 150px);
+    min-height: calc(50vh + 1200px);
+}
+
+content.center-second {
```
Comment (karakashevivo) [inline src/styles/styles.css:26]: It's not necessary to specify the html-element. It increase the specificity [long explanation](https://css-tricks.com/specifics-on-css-specificity/),  [short explanation](https://specifishity.com/). Instead use just the class selector `.center-second`

File: src/images/Fluiditype-logo.svg (hunk 1 above)
Comment (sboriskirov) [reply src/images/Fluiditype-logo.svg:8]: Done

**Pull request merged**